stable/stable_rec/network-stresser/templates/stresser-receiver-svc.yml

```diff
@@ -9,15 +9,15 @@ spec:
   ports:
   - name: tcp-receiver
     protocol: TCP
-    port: {{ .Values.cmdOptions.tcpPort }}
```

Comment:
Why you moved to fix values, can you change the defaults outside and not change to fix numbers ?